backend/telemetry_shard/src/aggregator.rs

```diff
@@ -246,7 +250,28 @@ impl Aggregator {
                     let _ = tx_to_telemetry_core
                         .send_async(FromShardAggregator::UpdateNode { local_id, payload })
                         .await;
-                }
+                },
+                ToAggregator::FromWebsocket(
+                    conn_id,
+                    FromWebsocket::Remove {
+                        message_id
+                    }
+                ) => {
+                    // Get the local ID, ignoring the message if none match:
+                    let local_id = match to_local_id.get_id(&(conn_id, message_id)) {
+                        Some(id) => id,
+                        None => continue,
+                    };
+
+                    // Remove references to this single node:
+                    muted.remove(&local_id);
+                    to_local_id.remove_by_id(local_id);
+
+                    // Tell the core to remove references to it, too:
+                    let _ = tx_to_telemetry_core
+                        .send_async(FromShardAggregator::RemoveNode { local_id })
+                        .await;
```

Comment:
Might be worth to put into its own utility method so we can rest assured none of these steps are ever skipped (I think it's repeated below as well?).

Reply:
I agree (and I tweaked the code so those 2 sections look identical); I'd need to restructure things a little to make that be the case though (or do something horrible like a macro), because those maps are just local variables, so I'm leaning towards not doing that right now.